doc/source/releases/23.1_announce_user.rst

```diff
@@ -0,0 +1,125 @@
+  `Pull Request 16187`_
+* Small test decorator improvements
+  (thanks to `@jmchilton <https://github.com/jmchilton>`__).
+  `Pull Request 16220`_
+
+New Datatypes
+===========================================================
+
+.. datatypes
+* Fix bad auto-merge of dev
+  (thanks to `@jmchilton <https://github.com/jmchilton>`__).
+  `Pull Request 15386`_
+* Follow up to `#15186 <https://github.com/galaxyproject/galaxy/issues/15186>`__
+  (thanks to `@jdavcs <https://github.com/jdavcs>`__).
+  `Pull Request 15388`_
+* Add support for visualizing HDF5 datasets
+  (thanks to `@jarrah42 <https://github.com/jarrah42>`__).
+  `Pull Request 15394`_
+* Merge 23.0 into dev
+  (thanks to `@mvdbeek <https://github.com/mvdbeek>`__).
+  `Pull Request 15594`_
+* Enable ``strict_equality`` mypy option
+  (thanks to `@nsoranzo <https://github.com/nsoranzo>`__).
+  `Pull Request 15808`_
+* FITS data format
+  (thanks to `@volodymyrss <https://github.com/volodymyrss>`__).
+  `Pull Request 15905`_
+* Improve display chunk generation for BAMs
+  (thanks to `@wm75 <https://github.com/wm75>`__).
+  `Pull Request 15972`_
+* Code cleanups from ruff and pyupgrade
+  (thanks to `@nsoranzo <https://github.com/nsoranzo>`__).
+  `Pull Request 16035`_
+* Dataset chunking tests (and small fixes)
+  (thanks to `@jmchilton <https://github.com/jmchilton>`__).
+  `Pull Request 16069`_
+* Merge release_23.0 into dev
+  (thanks to `@nsoranzo <https://github.com/nsoranzo>`__).
+  `Pull Request 16088`_
+* Improve robustness of collection edit E2E test
+  (thanks to `@jmchilton <https://github.com/jmchilton>`__).
+  `Pull Request 16092`_
+* Add hdf4 datatype
+  (thanks to `@TheoMathurin <https://github.com/TheoMathurin>`__).
+  `Pull Request 16105`_
+* Fix linting of FITS datatype code
+  (thanks to `@mvdbeek <https://github.com/mvdbeek>`__).
+  `Pull Request 16169`_
+* Add zipped mongodb and genenotebook datatypes
+  (thanks to `@abretaud <https://github.com/abretaud>`__).
+  `Pull Request 16173`_
+* Add Visium datatype for squidpy and spatialomics tools
+  (thanks to `@astrovsky01 <https://github.com/astrovsky01>`__).
+  `Pull Request 16255`_
+
+Builtin Tool Updates
+===========================================================
+
+.. tools
+* Deprecate tools/evolution
+  (thanks to `@bernt-matthias <https://github.com/bernt-matthias>`__).
+  `Pull Request 15656`_
+* Allow pending inputs in some collection operation tools
+  (thanks to `@mvdbeek <https://github.com/mvdbeek>`__).
+  `Pull Request 15892`_
+* Updates MGnify interactive tool to v1.1.0
+  (thanks to `@SandyRogers <https://github.com/SandyRogers>`__).
+  `Pull Request 16179`_
+* Remove various fallback behaviors
+  (thanks to `@mvdbeek <https://github.com/mvdbeek>`__).
+  `Pull Request 16199`_
+
+Release Testing Team
+===========================================================
+
+A special thanks to the release testing team for testing many of the new features and reporting many bugs:
```

Comment:
```suggestion
A special thanks to the release testing team for testing many of the new features:
```
Sometimes there may be not too many bugs, and even if there are many, reporting them is only part of what the team does. So maybe just limit it to thanks for testing new features?